test/featureselecturrule.jl

```diff
@@ -0,0 +1,63 @@
+using MLJBase
```

Comment:
the name of this file is incorrect and not at the right place + the whole test file should be a module like other test files, please look at `Transformer.jl` and either just put your code in there or imitate the style.